src/coreservices.cpp

```diff
@@ -466,10 +483,17 @@ void CoreServices::initialize(QApplication* pApp) {
 
     // Load tracks in args.qlMusicFiles (command line arguments) into player
     // 1 and 2:
+
     const QList<QString>& musicFiles = m_cmdlineArgs.getMusicFiles();
+
     for (int i = 0; i < (int)m_pPlayerManager->numDecks() && i < musicFiles.count(); ++i) {
         if (SoundSourceProxy::isFileNameSupported(musicFiles.at(i))) {
             m_pPlayerManager->slotLoadToDeck(musicFiles.at(i), i + 1);
+        } else if (isPlaylist(musicFiles.at(i))) {
+            qWarning() << "Detected Playlist file!!";
+            const QString playlistFile = musicFiles.at(i);
+            qWarning() << playlistFile;
```

Comment:
Please combine these log outputs and make it a qDebug(). The text can also be improved. 
"Found playlist file ... iin the command line arguments" or such.

Reply:
formatted in latest commit